eventarc/pubsub/README.md

```diff
@@ -0,0 +1,45 @@
+# Events for Cloud Run – Pub/Sub tutorial
+
+This sample shows how to create a service that processes Pub/Sub events using 
+[the CloudEvents SDK](https://github.com/cloudevents/sdk-python).
+
+## Quickstart
+
+Deploy your Cloud Run service:
+
+```sh
+gcloud builds submit \
+ --tag gcr.io/$(gcloud config get-value project)/eventarc-pubsub
+gcloud run deploy eventarc-pubsub \
+ --image gcr.io/$(gcloud config get-value project)/eventarc-pubsub
+```
+
+Create a Cloud Eventarc trigger, which will also create a Pub/Sub topic:
+
+```sh
+gcloud beta eventarc triggers create pubsub-trigger \
+--destination-run-service eventarc-pubsub \
+--matching-criteria "type=google.cloud.pubsub.topic.v1.messagePublished"
+```
+
+## Test
+
+Test your Cloud Run service by publishing a message to the topic: 
+
+```sh
+TOPIC=$(gcloud beta eventarc triggers describe pubsub-trigger \
```

Comment:
nit: add a comment about finding the topic name

Reply:
Done.